__tests__/get-articles-api.test.js

```diff
@@ -38,4 +38,41 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("responds with the article of the given query", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(articles.hasOwnProperty("body")).toBe(false);
+          expect(article.topic).toBe("mitch");

```

Comment:
Not a dealbreaker, but this test for topic being "mitch" can be inside the .toMatchObject, by expecting the value to be mitch instead of an expect.any(String)